spec/tests/database/nosql/mongodb/mongod_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'spec_helper'
+#
+# cloud::database::nosql::mongodb::mongod
+#
+
+# Since mongod could be run in three different modes:
+# 
+#  * Normal listening on 27017
+#  * Shardsvr listening on 27018
+#  * Confisvr listening on 27019
+#
+# We make sure the mongod process is running and not
+# which port it is listening on
+
+describe process('mongod') do
```

Comment:
Yes, 'mongod' is the name of the process on both OS

On Debian : 

```
root@vagrant:/home/vagrant# cat /etc/debian_version 
7.3
root@vagrant:/home/vagrant# ps fauxwww | grep mongo
root      3207  0.0  0.1   7828   840 pts/0    S+   07:05   0:00                          \_ grep mongo
mongodb   3180 19.6  7.6 187508 38792 ?        Sl   07:04   0:08 /usr/bin/mongod --config /etc/mongod.conf
root@vagrant:/home/vagrant# 
```

On RedHat : 

```
[root@vagrant]# cat /etc/redhat-release 
Red Hat Enterprise Linux Server release 7.0 (Maipo)
[root@vagrant]# ps fauxwww | grep mongo
mongodb  45175  0.6  0.4 5492852 71884 ?       Sl   Feb04   5:49 /usr/bin/mongod --quiet -f /etc/mongodb.conf run
root     23682  0.0  0.0 112640   960 pts/1    S+   15:06   0:00                          \_ grep --color=auto mongo
[root@vagrant]# 
```